docs/4.x/tracking-javascript-guide.md

```diff
@@ -796,11 +796,7 @@ If you want to ignore download or outlink tracking on a specific link, you can a
 If you'd like to provide your users with the ability to opt-out entirely from tracking, you can use an opt-out form. Matomo ships with an
 opt-out form implementation that uses third-party cookies (which you can configure within Matomo on the _Matomo > Administration > Privacy_ page).
 
-This form is simple to embed since it only requires that you add an iframe to your website, but it is not always ideal. Some users block third-party
-cookies so the opt-out form wouldn't work for them. You may also want to display custom text or graphics in the opt-out form, or you may
-want to allow users to opt-out of your sites individually instead of altogether.
-
-In such a case, you may want to consider creating a custom opt-out form. The specifics of creating an HTML/JS form are out of scope for
+This form is simple to embed since it only requires that you add an iframe to your website, but it is not always ideal. In such a case, you may want to consider creating a custom opt-out form. The specifics of creating an HTML/JS form are out of scope for
```

Comment:
Guess it actually no longer does require adding an iframe, but some javascript instead? 🤔

Reply:
@bx80 can you please suggest an accuracy improvement on my sentence here?

Reply:
Perhaps something like:

```suggestion
This form is simple to embed since it only requires that you add a snippet of code to your website, but it is not always ideal. In such a case, you may want to consider creating a custom opt-out form. The specifics of creating an HTML/JS form are out of scope for
```